fixtures/sum_kasiopea/gen.cpp

```diff
@@ -1,6 +1,8 @@
 #include <iostream>
+#include <string.h>
 #include <string>
 #include <random>
+#include <cassert>
```

Comment:
Why does one import of a C header use the `[name].h` format, while another uses the `c[name]` format?